### PR TITLE
#CP-24 feat: Support multiple language

### PR DESCRIPTION
**Pull request by feyton**
### What the PR does:
The PR add support of multiple languages and configure Axios to send the `Accepted-Language` header on every request.
- [x] The project is set to use multiple language
- [x] The selected language is being sent on each request

### Description of the tasks to be completed:

- [x] The user is able to select and change the language
- [x] The landing page is fully translated
- [x] The axios is setup to send request with the header respecting the selected language
- [x] The user selected language is saved so that next time they will use it

### How should this be manually tested
After creating an empty directory on your machine, 
1. Open the terminal at the directory level (Empty)
2. Copy the commands bellow and paste into the terminal

```
git clone https://github.com/atlp-rwanda/phantom-fe-codebandits .
git fetch origin 
git checkout ft-multi-language-support
npm install
npm start
open http://localhost:3000
// Follow the link opened in the default browser
```
OR go to [http://phantom-codebantis.herokuapp.com/](http://phantom-codebantis.herokuapp.com/)

### Why should this be merged ASAP:
Since the integration is starting,to take full advantage of the translations on the backend this Feature is critical for that.

### CODE REVIEW

**Review by samyshyaka — DISMISSED** [review]
Nice work @feyton, this is very useful because it will allow us to make translations for multiple features of this application and thus we will be able to increase it's accessibility.

**Review by Yvad60 — DISMISSED** [review]
Nice work @feyton with the translations on the front end. I have noticed that in some cases translations in other languages break page designs maybe because of words length maybe we will try to find a work around that.  for now this PR is ready for merge

File: src/pages/LandingPage.js [hunk 1]
```diff
@@ -1,49 +1,45 @@
...
-              />
-            </div>
+      <div>
+        <Intro />
+        <div className="flex flex-col items-center">
+          <h1 className="text-4xl font-raleway font-extrabold mt-28 xl:mt-44 mb-10 text-center">
+            {t('how_it_work')}
+          </h1>
+          <div className=" flex flex-col lg:flex-row xl:flex-row mt-12 mb-32">
+            <Card
```
Comment (Jkadhuwa) [inline src/pages/LandingPage.js]: We can create a data object and loop it through the card component with the data. that will solve the codeclimate issue

**Review by Jkadhuwa — DISMISSED** [review]
Good work, left some comment for you to attend to.

**Review by samyshyaka — APPROVED** [review]

**Review by Jkadhuwa — APPROVED** [review]

**Pull request merged**